tutorials/best_practices/scene_organization.rst

```diff
@@ -65,7 +65,7 @@ initialize it:
      .. code-tab:: csharp
 
        // Parent
-       GetNode("Child").Connect("SignalName", ObjectWithMethod, "MethodOnTheObject");
+       Callable.From(ObjectWithMethod.MethodOnTheObject);
```

Comment:
This is not correct, you're not connecting to anything now, you're just creating a callable, it should be:
```suggestion
      GetNode("Child").Connect("SignalName", Callable.From(ObjectWithMethod.MethodOnTheObject));
```
I believe